### ISSUE TITLE
Cannot compile with `create-react-app`

### ISSUE THREAD

**Issue by shantur** [js-rcon/react-console-emulator#411]
Due to `stringify-object` being dependent on `get-own-enumerable-property-symbols` and  target for Node.js instead of browsers ( https://github.com/yeoman/stringify-object/issues/49#issuecomment-305090383 )
UglifyJS throws error while compiling this with react
```
SyntaxError: Unexpected token: operator (>) [./~/get-own-enumerable-property-symbols/lib/index.js:3,0]
```

Just a thought why not use `JSON.stringify` instead?

**Comment by linuswillner** [comment]
That shouldn't be in there anymore... Will investigate.

**Comment by shantur** [comment]
Ignore me false alarm, i just found out CRA version was too low.

**Comment by linuswillner** [comment]
@shantur Don't apologise, because you did point me to an actual legitimate issue here! The string comparison stuff was not meant to be there anymore; it was supposed to have been removed in 3.0.2 due to issues reported in #35. I've removed the leftover `stringify-object` logic and fixed the comparison issues in v3.0.3, which I just pushed. Thanks for the report!

**Comment by shantur** [comment]
Thanks @linuswillner
